src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -177,28 +177,41 @@ Follow these guidelines when you use `<var>` tags:
+  * [Instrumentation procedure](/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/enable-new-relic-monitoring-aws-lambda#java) (at bottom of the Java section): the var tag wouldn't be necessary because it's an example, not part of a procedure, and the main goal is seeing the general structure. Also, because it's an example of app code, the concept of user-specific values doesn't have much meaning, because the entire code will vary dependent on how the customer has written their code. (If anything, this would be a potential for using [`<mark>` format](#highlighting) to emphasize the New Relic functions.)
+* Response/output code: If the code is meant to show an expected return, and is not related to a procedure, then var tags shouldn't be used. Here's [a doc section](/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api#get-all-monitors) (the returned JSON) where var tags are not needed.
+
+For some use cases, [highlighting](#annotate-mark) may be a better choice than a `var` tag.
+
 <CollapserGroup>
   <Collapser
     id="var-tag-examples"
-    title="Examples of using the <var> tag"
+    title="Examples of `<var>` tag use"
   >
-  ```
-  echo "license_key: <var>YOUR_LICENSE_KEY</var>" | sudo tee -a /etc/newrelic-infra.yml
-  ```
-  
-  ```
-  <var>PATH/TO/SOMETHING.exe</var>
-  ```
-  
-  ```
-  msiexec.exe /qn /i <var>PATH\TO</var>\newrelic-infra.msi
-  ```
+Below are some general style recommendations for formatting user-specific `<var>` values. `<var>` tag formatting may vary based on language- or system-specific expectations, so be sure check the style used in the documentation section in question, and to ask relevant SMEs what they think of the style.
+
+* Account IDs and other IDs/#s: `<var>YOUR_ACCOUNT_ID</var>`, `<var>YOUR_API_KEY</var>`, etc. **This should be the general style used.**
+* URLs: [example.com](https://example.com), or maybe <var>`YOUR_URL`</var>
+* Paths: `<var>PATH/TO/SOMETHING.exe</var>` or maybe `<var>PATH_TO_FILE</var>`
+* Emails: [datanerd@example.com](mailto:datanerd@example.com) or maybe `<var>YOUR_EMAIL</var>`
+* Bash variables for REST API code: Some `<var>` tagged code values on the docs site have the form $&#x7B;API_KEY}. This is a format used for variables in bash scripts, where users assign values to specific variable names and then call those variables later in the script by using the $VARIABLE_NAME. For more info, see this [explanation of bash variables](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-5.html).
+
+  <Callout variant="important">
+    The bash variable style is currently used in the REST API docs and in some Synthetics docs and that's fine. But going forward we should use the general variable style (without the $ and `&lt;` >).
+  </Callout>
```

Comment:
Not super fan if this. Suggestion: The bash variable style is currently used in the REST API docs, and in some Synthetics docs. However, going forward we should use the general variable style, without the $ and `&lt;` >.